consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/IbftHelpers.java

```diff
@@ -51,4 +55,28 @@ public static Block createSealedBlock(
 
     return new Block(sealedHeader, block.getBody());
   }
+
+  public static Optional<PreparedCertificate> findLatestPreparedCertificate(
```

Comment:
doesn't look there is a test for this

Reply:
Was previously fully tested as part of NewRoundMessageValidator (but was then moved out here) - have created additional tests based of the previously existing tests.